lib/glific/flows.ex

```diff
@@ -368,7 +368,7 @@ defmodule Glific.Flows do
   which is an array of objects in the json file. Used for Node/Actions, Node/Exits,
   Router/Cases, and Router/Categories
   """
-  @spec build_flow_objects(map(), map(), (map(), map(), any -> {any, map()}), any) ::
+  @spec build_flow_objects(list(), map(), (map(), map(), any -> {any, map()}), any) ::
```

Comment:
@AkhileshNegi  I think we need to revert this as well.

Reply:
@pankaj-ag No. I checked and it was wrong spec for the function. Wondering why mix check didn't give any error